widgetssdk/src/main/java/com/glia/widgets/core/chathead/ChatHeadService.java

```diff
@@ -93,11 +94,18 @@ private void initChatHeadView(
         int screenWidth = displayMetrics.widthPixels;
 
         params.gravity = Gravity.TOP | Gravity.START;
-        params.x = getDefaultXPosition(screenWidth);
-        params.y = getDefaultYPosition(screenHeight);
+
+        Pair<Integer, Integer> savedPosition = controller.onChatHeadPositionRequested();
```

Comment:
here `saved` doesn't add useful info, better just `chatHeadPosition`. Whether it's saved or not, how saved etc is controller business.